sdk/core/Azure.Core/src/RequestFailedException.cs

```diff
@@ -172,55 +173,64 @@ private static ErrorDetails CreateExceptionDetails(Response response, RequestFai
                 .Append("Status: ")
                 .Append(response.Status.ToString(CultureInfo.InvariantCulture));
 
-            if (!string.IsNullOrEmpty(response.ReasonPhrase))
+            if (response.IsError)
             {
-                messageBuilder.Append(" (")
-                    .Append(response.ReasonPhrase)
-                    .AppendLine(")");
-            }
-            else
-            {
-                messageBuilder.AppendLine();
-            }
+                if (!string.IsNullOrEmpty(response.ReasonPhrase))
```

Comment:
Why not include ReasonPhrase with a success result?

Reply:
I debated whether to include it - what is a typical value for success?

Reply:
I think it's part of the HTTP spec, so it seems strange to remove.  It's like 200: "OK", 404: "NotFound", stuff like that.

Reply:
But if you know of a case where services rewrite it with sensitive content, that would be a reason to remove it.  I just don't know of one.

Reply:
makes sense, I'll add it back.